scheduler/cloudformation/cloudformation.go

```diff
@@ -414,16 +510,19 @@ func (s *Scheduler) updateStack(ctx context.Context, input *updateStackInput, do
 //   until the other stack operation has completed.
 // * If there is another pending stack operation, it will be replaced by the new
 //   update.
-func (s *Scheduler) performStackOperation(ctx context.Context, stackName string, fn func() error, waiter func(*cloudformation.DescribeStacksInput) error, ss scheduler.StatusStream) error {
+func (s *Scheduler) performStackOperation(ctx context.Context, stackName string, fn func() error, waiter func(*cloudformation.DescribeStacksInput) error, ss scheduler.StatusStream) stackOperationOutput {
```

Comment:
I think we should probably still make the return value of this `(*cloudformation.Stack, error)`, then wrap it in a `stackOperationOutput` when we send it on the channel.